python/py-ldap/Portfile

```diff
@@ -17,26 +19,46 @@ long_description \
                 Additionally the package contains modules for other \
                 LDAP-related stuff (e.g. processing LDIF, LDAPURLs, \
                 LDAPv3 schema, etc.).
-
 homepage        http://www.python-ldap.org/
-python.rootname python-ldap
-checksums       rmd160 846bf38cb166f478f2cf401f889ad7ecd39b05c5 \
-                sha256 7d1c4b15375a533564aad3d3deade789221e450052b21ebb9720fb822eccdb8e \
-                size   367645
 
-python.versions 27 36 37
+checksums       rmd160  094d82b4e4dc80d7a11083254197cfc16b84832e \
+                sha256  b16470a0983aaf09a00ffb8f40b69a2446f3d0be639a229256bce381fcb268f7 \
+                size    378058
+
+python.versions 27 36 37 38 39 310
```

Comment:
can PY36 be removed?

Reply:
Based on yesterday's checks, there don't appear to be any dependents on 3.6... so technically it can be removed.

But I figured that since we're supporting 2.7 anyway - necessitated by at least one dependent - I was reluctant to remove support for older versions.